Automattic/jetpack
# Add/my home to classic view

Fixes # p1710554857070379/1709722568.272419-slack-C06DN6QQVAQ

## Proposed changes:

* Adds a "My Home" link for Jetpack sites in classic view nav unification.
* 'default' view should be unaffected
* 'classic' view + ( proxy OR early release sticker): should be unaffected.

After:

<img width="594" alt="Screenshot 2024-03-18 at 8 32 32 AM" src="https://github.com/Automattic/jetpack/assets/28742426/46773215-c6e5-42d2-a1b3-fce02bc35e98">
<img width="444" alt="Screenshot 2024-03-18 at 8 32 44 AM" src="https://github.com/Automattic/jetpack/assets/28742426/9cf2d631-394e-4fe6-9baa-bb4723fc986d">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Disable proxy, as this code will not run while proxied due to the untangling project.
* Enable this branch through the jetpack beta plugin on an atomic site in 'classic' view.
* verify a "my home" menu item appears below the site card and works as expected.
